### PR TITLE
Upgrade to Akka 2.4

### PR DESCRIPTION
**Pull request by paulcleary**
Most of the changes are in the build setup.  We are no longer using an external POM file to manage sirius, purely SBT.

### CODE REVIEW

**Comment by joercampbell** [comment]
Just out of curiosity - This akka upgrade isn't breaking like the last one (from a binary message perspective between actors) is it?

**Comment by paulcleary** [comment]
@joercampbell this is a WIP only; just wanted to post it so others can take a look.  Told PC2 I would post it to see what changed.

We would definitely need to do some backward compatibility testing.  There is quite a roadmap that we would need to go through before this can be merged.  That is something PC2 and I will discuss in the coming week or so. 

So far, this has only been tested in a multi-node cluster for a new implementation.

File: project/Build.scala [hunk 1]
```diff
@@ -1,57 +0,0 @@
...
-      (binFile) =>
-        JRuntime.getRuntime.exec("chmod 755 " + binFile)
-    )   
-    JRuntime.getRuntime.exec("chmod 755 " + (binDir / "waltool").getPath)
-    JRuntime.getRuntime.exec("chmod 755 " + (binDir / "nodetool").getPath)
-
-    JRuntime.getRuntime.exec("tar -C target -czf target/sirius-standalone.tgz sirius-standalone")
-  }
-  
-}
```
Comment (clinedome-work) [inline project/Build.scala]: I think we probably want to keep this. We don't want to get rid of the `sirius-standalone` package.

**Comment by clinedome-work** [comment]
@paulcleary a couple things: one, can you change the `.travis.yml` to hit `2.11.8` instead of `2.10.2`. Two, please undelete the `Build.scala` file so we can build the standalone artifact. 

@joercampbell unfortunately, I think the upgrade from scala `2.10.2` to `2.11.8` does introduce serialization changes that will cause mixed-version clusters to have problems during a rolling upgrade. Same as the last major upgrade, we'll need to account for this fact in the release notes, and make sure folks know as they go into the upgrade process. I don't think there's a short away around it for this.

**Comment by paulcleary** [comment]
@clinedome-work I made the adjustments.

**Pull request merged**